usage-examples/clm-for-hw-sw.md

```diff
@@ -0,0 +1,62 @@
+<!---
+   Copyright 2023 Ericsson AB.
+   For a full list of individual contributors, please see the commit history.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+--->
+
+# Confidence Level for a Combination of Hardware and Software
+
+This example shows how to use
+[EiffelConfidenceLevelModifiedEvent (CLM)](../eiffel-vocabulary/EiffelConfidenceLevelModifiedEvent.md)
+and [EiffelCompositionDefinedEvent (CD)](../eiffel-vocabulary/EiffelCompositionDefinedEvent.md)
+for describing confidence over hardware and software artifacts.
+
+## Introduction
+
+Eiffel has the ability to support a continuous integration and delivery pipeline
+for hardware products containing software. When developing the software for the
+hardware, the designer first runs test specifically for the software part. The
+designer then loads the software on the hardware and runs test with both the
+hardware and software. These tests produce the confidence signaled

```

Comment:
> and runs test with both the

Should this be "and runs tests with" or "and runs a test"? Probably the former given "These tests" on the following line.

Reply:
Fixed in [40837c5](https://github.com/eiffel-community/eiffel/pull/369/commits/40837c54335915fba58f5f91afb2569bb8cd5437)